### ISSUE TITLE
Failed to open file (exit code 3221225781)

### ISSUE THREAD

**Issue by tr33ox** [mifi/lossless-cut#927]
when I try to open any file, I receive this error:

>  Failed to open file
> Command failed with exit code 3221225781: C:\Users\****\Desktop\LosslessCut-win\resources\node_modules\ffmpeg-ffprobe-static\ffprobe.exe -of json -show_format -i C:\Users\****\Desktop\test.mp4 -hide_banner

![Screenshot 2021-11-13 032058](https://user-images.githubusercontent.com/94205360/141598782-e4ef607f-3146-4af5-9342-7655b3c8ed86.png)

and when I press on "Report an error", it doesn't show any problem as follows:
> No error
> 
> {
>   "state": {
>     "filePath": "",
>     "externalStreamFiles": [],
>     "mainStreams": [],
>     "copyStreamIdsByFile": {},
>     "cutSegments": [
>       {}
>     ],
>     "rotation": 360,
>     "shortestFlag": false
>   },
>   "platform": "win32",
>   "version": "3.39.0"
> }

I think this problem indicates missing libraries, but I couldn't fix it.

**Comment by mifi** [comment]
Hi. could you try to run ffprobe.exe from the command prompt and see if you get any error there?
also which OS are you using? maybe you're missing some OS components like VC++ distributable
https://stackoverflow.com/questions/49413443/trouble-debugging-error-with-exit-code-3221225781-missing-libraries-in-windows

**Comment by tr33ox** [comment]
Hi. thank you very much, your reply helped me fix the problem.
actually, I did install VC++ distributable when I saw the problem for the first time, but to no avail.
anyway, this was the key:
> run ffprobe.exe from the command prompt and see if you get any error there
when I ran ffprobe.exe I got this error:
![Screenshot 2021-11-13 201641](https://user-images.githubusercontent.com/94205360/141653350-90dfdee3-23a8-4e52-9e1e-22d32b082339.png)
now it is clear what I am missing! so I downloaded MFPlat.DLL, and tried again
ffprobe.exe --> working
LosslessCut -->no problem
